src/snyk/common/services/downloadService.ts

```diff
@@ -105,11 +103,17 @@ export class DownloadService {
       return false;
     }
     const latestChecksum = await this.cliApi.getSha256Checksum(version, platform);
-    const path = await CliExecutable.getPath(
-      this.extensionContext.extensionPath,
-      await this.configuration.getCliPath(),
-    );
-
+    const path = await this.configuration.getCliPath();
+    // migration for moving from default extension path to xdg dirs.
+    if (CliExecutable.isPathInExtensionDirectory(this.extensionContext.extensionPath, path)) {
```

Comment:
this might be unnecessary. But I want to ensure extension doesn't break for current preview users.